src/database/models/team_item.rs

```diff
@@ -27,44 +28,48 @@ impl TeamBuilder {
     ) -> Result<TeamId, super::DatabaseError> {
         let team_id = generate_team_id(&mut *transaction).await?;
 
-        let team = Team { id: team_id };
+        let team = Team {
+            id: team_id,
+            association: self.association_id,
+        };
 
         sqlx::query!(
             "
-            INSERT INTO teams (id)
-            VALUES ($1)
+            INSERT INTO teams (id, project_id, organization_id)
+            VALUES ($1, $2, $3)
             ",
             team.id as TeamId,
+            match team.association {
+                TeamAssociationId::Project(pid) => Some(pid.0),
+                TeamAssociationId::Organization(_) => None,
+            },
+            match team.association {
+                TeamAssociationId::Project(_) => None,
+                TeamAssociationId::Organization(oid) => Some(oid.0),
+            },
```

Comment:
does team association need to be kept track of in the api? don't see the point. might be wrong tho

Reply:
Kept the structure as its useful, but no longer in the db as discussed on disc